github-webhook-handler.js

```diff
@@ -76,6 +76,8 @@ function create (options) {
           event   : event
         , id      : id
         , payload : obj
+        , protocol: req.protocol
+        , host    : req.get('host')
```

Comment:
My bad, I was thinking of ExpressJs where you can actually do this. req.headers['host] is probably the better way to go.